.github/workflows/test-python.yml

```diff
@@ -55,18 +52,16 @@ jobs:
       - uses: extractions/setup-just@v2
       - name: Start MongoDB on Linux
         if: ${{ startsWith(runner.os, 'Linux') }}
-        run: |
-            docker run --name mongodb -p 27017:27017 -e MONGO_INITDB_DATABASE=unittest --detach mongo:${MONGODB_VERSION} mongod --replSet rs --setParameter transactionLifetimeLimitSeconds=5
-            until docker exec --tty mongodb mongosh 127.0.0.1:27017 --eval "db.runCommand({ ping: 1 })"; do
-              sleep 1
-            done
-            sudo docker exec --tty mongodb mongosh 127.0.0.1:27017 --eval "rs.initiate({\"_id\":\"rs\",\"members\":[{\"_id\":0,\"host\":\"127.0.0.1:27017\" }]})"
+        uses: supercharge/mongodb-github-action@1.12.0
+        with:
+          mongodb-version: 8.0
+          mongodb-replica-set: test-rs
       - name: Start MongoDB on MacOS
         if: ${{ startsWith(runner.os, 'macOS') }}
         run: |
           brew tap mongodb/brew
-          brew install mongodb/brew/mongodb-community@${MONGODB_VERSION}
-          brew services start mongodb-community@${MONGODB_VERSION}
+          brew install mongodb/brew/mongodb-community@7.0
```

Comment:
How come this change?

Reply:
This is the highest version supported on brew right now, but not the highest available with the GitHub action on Ubuntu.

Reply:
Actually, I must have had it misconfigured before, it works with 8.0.